filebuffer.go

```diff
@@ -32,6 +33,16 @@ func New(b []byte) *Buffer {
 	return &Buffer{Buff: bytes.NewBuffer(b)}
 }
 
+// NewFromReader is a convenience method that returns a new populated Buffer
+// whose contents are sourced from a supplied reader.
```

Comment:
Would you mind adding a small note that the source will be entirely read in memory (instead of being streamed). This might be surprising for some users and can will use more memory depending on the source.

Reply:
Wow, how did a week go by already? Great point. I've added this.